website/src/components/Property.tsx

```diff
@@ -54,7 +54,9 @@ type SingleTypeOrValue =
   | {
       type: 'custom';
       name: string;
-      generics?: TypeOrValue[];
+      generics?:
+        | { withDefault: true; name: string; type: TypeOrValue }
```

Comment:
Can we rename that to `default` so that it is similar to the `default` prop of the component?

Reply:
My idea would be to change it to `default?: TypeOrValue`.

Reply:
Maybe your approach is the right one. I will probably take a closer look tomorrow.